addons/website_slides_sale/views/slide_channel_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="ir.ui.view" id="view_slide_channel_form_inherit_website_slides_sale">
```

Comment:
id first.
`id="slide_channel_view_form"`. 'inherit' is not mandatory anymore, as we are in a bridge module, the prefix will tell us this is a view extension.

Reply:
Done.